secret-sender.rb

```diff
@@ -0,0 +1,25 @@
+require 'fileutils'
+
+class SecretSender < Formula
+  desc 'SecretSender is a little utility to securely share secrets to other users.'
+  homepage 'https://github.com/Shopify/secret-sender'
+  url 'https://github.com/Shopify/secret-sender/archive/v1.0.0.tar.gz'
+  sha256 'f395b5da63d6c031b756622b6ad8430e8ca54d60d8d7266d7132b1265c4e2d39'
+
+  depends_on 'go' => :build
+
+  bottle do
+    root_url "https://s3.amazonaws.com/burkelibbey"
```

Comment:
Why is the bottle hosted on your s3? Can you attach it to the GitHub release instead?

Reply:
Maybe, I wonder if that works...